lib/main.dart

```diff
@@ -22,13 +23,13 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final appTitle = 'Project Eureka';
+    //final Color ourBlueBackground = Color(0xFF37474F);
```

Comment:
Remove comments if you aren't using them

Reply:
I agree with Tony